components/create_comment/create_comment.jsx

```diff
@@ -548,7 +559,29 @@ export default class CreateComment extends React.PureComponent {
         this.draftsForPost[this.props.rootId] = updatedDraft;
     }
 
+    handleMouseUp = (e) => {
+        e.persist();
+        const caretPosition = Utils.getCaretPosition(e.target);
+        const target = e;
+        this.setState({
+            caretPosition,
+            target,
+        });
+    }
+
     handleKeyDown = (e) => {
+        e.persist();
+
+        // A bit of a hack. Better to use keyup instead.
```

Comment:
Would it be difficult to add a keyup handler for this? Using `setTimeout` like this seems like it coudl cause other issues later on

Reply:
@hmhealey I actually wanted to create one but thought to reuse an already existing event handler was appropriate. I have now added a keyup handler. Thanks for the review!